internal/rpc/rpcserver/server.go

```diff
@@ -2050,7 +2050,7 @@ func (s *walletServer) ValidateAddress(ctx context.Context, req *pb.ValidateAddr
 	switch ka := ka.(type) {
 	case wallet.BIP0044Address:
 		_, branch, child := ka.Path()
-		result.IsInternal = branch == 1
+		result.IsInternal = branch == udb.InternalBranch
```

Comment:
For future reference, udb should not be used anywhere outside of the wallet package.  This is a layering violation and it's making it more difficult to convert the package to being internal (but this particular case is not as bad as say, depending on a type defined by that package).